modules/ROOT/pages/configure-message-flows.adoc

```diff
@@ -82,15 +83,15 @@ This is the target message type and the Endpoint where you want to route the tra
 . Either search and select an existing message type, or create a new message type. To create a new message type, depending on the message standard, follow instructions in one of these sections.
 
 +
-* xref:document-types.adoc[Message (Document) Types]
+* xref:document-types.adoc[Message Types]
 +
 .  Click *Select* to set up a target endpoint. Either search and select and existing endpoint, or create a new endpoint.
```

Comment:
Is this one step or two? Does the user click *Select* once or twice?
From what location does the user click *Select* in the first sentence?
```suggestion
.  Click *Select* to set up a target endpoint. 
. Either search for and select an existing endpoint, or create a new endpoint by clicking *New*.
```
?